collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -320,16 +319,16 @@ public void allowSwiping(boolean doSwipe) {
      */
     private boolean locationPermissionsPreviouslyGranted;
 
-    FormEntryViewModel viewModel;
+    private FormEntryViewModel formEntryViewModel;
```

Comment:
Probably this can be renamed to `BackgroundLocationViewModel`, now.

Reply:
Yeah totally agree.